index.js

```diff
@@ -1 +1,25 @@
-// JS code goes here
\ No newline at end of file
+// JS code goes here
+const hamburger = document.querySelector('.hamburger')
+const mobnav = document.querySelector('.mob__nav__menu')
+const nvsection = document.querySelector('.nv__section')
+const hamburger2 = document.querySelector('.mob__nav__menu .hamburger2')
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active')
+  mobnav.classList.toggle('active')
+  nvsection.classList.toggle('active')
+})
+
+document.querySelectorAll('.nv__item').forEach((n) =>
+  n.addEventListener('click', () => {
```

Comment:
- [ ] You did a great job with this implementation, and to completely make it the masterpiece that I think it currently is, kindly avoid the use of arbitrary or generic names. As professionals with the mindset of maintainability or collaboration, it is best to use descriptive naming techniques for our `variables`, `parameter` or `arguments` in functions. For this, do you think you can use a more descriptive word like `item`, `link` or `nav` instead of the generic `n` that you have here? This will make your code more readable and make working with others less stressful. 💪